person.rb

```diff
@@ -0,0 +1,19 @@
+# This is the Person class, parent of student and teacher class
+class Person
+  def initialize(age, id = 0, name = 'Unknown', parent_permission: true)
+    @id = id
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
```

Comment:
- [OPTIONAL] What if we generate a random id for our Person class 🤔 This might look like this:

```ruby
  def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```

Reply:
Solved thank you